qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/settings/capabilities/ServerCapabilitiesConstants.java

```diff
@@ -68,5 +71,20 @@ private ServerCapabilitiesConstants() {
 	public static final CompletionOptions DEFAULT_COMPLETION_OPTIONS = new CompletionOptions(false,
 			Arrays.asList("{", "@", "#", ".", ":"));
 	public static final DocumentLinkOptions DEFAULT_DOCUMENT_LINK_OPTIONS = new DocumentLinkOptions(true);
+	public static final CodeActionOptions DEFAULT_CODE_ACTION_OPTIONS = new CodeActionOptions();
+	static {
+		List<String> allCodeActionKinds = Arrays.asList( //
+				CodeActionKind.QuickFix, //
+				CodeActionKind.Empty, //
+				CodeActionKind.Refactor, //
+				CodeActionKind.RefactorExtract, //
+				CodeActionKind.RefactorInline, //

```

Comment:
Are you sure that we support all CodeActionKind like refactor, etc 

Today we have just QuickFix

Reply:
Okay. I'll remove everything except QuickFix and Empty